frontend/src/lib/components/LLMButton/ExplainCSPViolationButton.tsx

```diff
@@ -14,13 +16,16 @@ export const ExplainCSPViolationButton = ({
     ...buttonProps
 }: ExplainCSPViolationButtonProps): JSX.Element => {
     const [loading, setLoading] = useState(false)
+    const [isOpen, setIsOpen] = useState(false)
```

Comment:
Could've been only one, but I felt weird to do `setDescription('')` to close the modal.